encrypt/encrypt.go

```diff
@@ -75,9 +75,14 @@ func Encrypt(args []string) error {
 		return err
 	}
 
-	// Exit if public key is not provided
+	// no key provided, check for one in the session file
 	if len(publicKeyFileList) == 0 {
-		return fmt.Errorf("public key not provided")
+
+		sesKey, err := helpers.GetPublicKey()
+		if err != nil {
+			return fmt.Errorf("public key not provided, details: %v", err)
```

Comment:
```suggestion
			return fmt.Errorf("public key not provided or %v", err)
```
so that the error msg applies covers also the case when a user intends to input a public key in the command line.